### DISCUSSION TITLE
Unable to completely disable form step buttons on a step

### DISCUSSION THREAD

**Discussion by maxwellsmart84** [vueform/vueform#158] [Questions]
It doesn't look like we have the ability to completely disable buttons on a form step - not sure if this is a bug or by design.

[Link to Sandbox](https://codesandbox.io/p/devbox/serverless-bird-5v99mw?file=%2Fsrc%2FApp.vue)

**Comment by adamberecz** [comment]
It seems like the Finish button could indeed not be disabled, which got fixed in https://github.com/vueform/vueform/commit/831f7fdb42966b43d73c4e02d212bcc37ea32124.

If you completely want to get rid of step buttons and eg. use custom ones, you can just leave them out from the template:
```vue
<Vueform>
  <template #empty>
    <FormSteps>
      <FormStep name="test" label="Test" :elements="['hello_world']" />
    </FormSteps>
    <FormElements>
      <TextElement name="hello_world" label="Hello" placeholder="World" />
    </FormElements>
    <!-- <FormStepsControls /> -->
  </template>
</Vueform>
```

**Reply by softail** [reply]
How can i do it using schema?
